frontend/src/component/Account/PersonalData.styl

```diff
@@ -1,41 +1,37 @@
 @import "~tracim_frontend_lib/src/css/Variable.styl"
 
 .personaldata
-  &__text
-    margin 15px 0
-    font-size 18px
   &__form
-    margin 25px 0
     &__txtinput
       width 300px
       border-width 1px
       border-style solid
-      border-radius 5px
+      border-radius borderRadiusStandard
       &[placeholder]
         white-space nowrap
         overflow hidden
         text-overflow ellipsis
       &:disabled
         cursor not-allowed
       &.withAdminMsg
-        margin-right 20px
+        margin-right standardSpacing
       &__info
-        margin 5px
+        margin textSpacing
         &__icon
-          margin-right 5px
+          margin-right textSpacing
       &__msgerror
         width 300px
-        font-size small
+        font-size fontSizeMetadata
```

Comment:
Only for me, what is difference between fontSizeMetadata and fontSizeSmall ?

Reply:
It's a new naming, more connected to the semantics of its use than to its visual characteristics